examples/impute/plot_iterative_imputer_variants_comparison.py

```diff
@@ -57,6 +57,9 @@
 rng = np.random.RandomState(0)
 
 X_full, y_full = fetch_california_housing(return_X_y=True)
+# ~2k samples is enough for the purpose of the example.
```

Comment:
Add "Remove the following two lines for a slower run with different error bars". or something

Reply:
Done.